src/main/java/com/softserve/academy/spaced/repetition/repository/CourseRepository.java

```diff
@@ -43,4 +44,8 @@
     Set<BigInteger> findCoursesId(@Param("searchString") String searchString);
 
     List<Course> findByNameIgnoreCaseContainingOrDescriptionIgnoreCaseContaining(String name, String description);
+
+    @Modifying
+    @Query(value = "DELETE FROM user_courses WHERE course_id = :courseId", nativeQuery = true)
```

Comment:
I am not an expert in Hibernate but shouldn't we avoid usage of native querys? Can you explain why you used it here?

Reply:
You are right, we should avoid native queries syntax. But in this case, we can't do it because we have @ManyToMany on courses side and @OneToOne on users side relationship. It's not the right approach. So we have to change @OneToOne to @ManyToMany with the following change all parts of code of project depend on this relationship or write native query. I decided use second one.

Reply:
As long as it follows the ANSI SQL standard it is ok.